igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/client/model/v3/V3Build.groovy

```diff
@@ -76,4 +76,12 @@ class V3Build {
     boolean spinnakerTriggered(){
         return (eventType == "api" && commit.message == "Triggered from spinnaker")
     }
+
+    public String toString() {
+        String tmpSlug = "unknown/repository"
```

Comment:
`String tmpSlug = repository == null ? "unknown/repository" : repository.slug`

Reply:
I'll leave it as it is. I have plans to migrate the code to java soon.